flutter/lib/src/screenshot/recorder.dart

```diff
@@ -82,12 +90,21 @@ class ScreenshotRecorder {
       final picture = recorder.endRecording();
 
       try {
-        final finalImage = await picture.toImage(
-            (srcWidth * pixelRatio).round(), (srcHeight * pixelRatio).round());
+        Image finalImage;
+        if (config is ScheduledScreenshotRecorderConfig) {
+          finalImage = await picture.toImage((srcWidth * pixelRatio).round(),
+              (srcHeight * pixelRatio).round());
+        } else {
+          final targetHeight = config.quality
+              .calculateHeight(srcWidth.toInt(), srcHeight.toInt());
+          final targetWidth = config.quality
+              .calculateWidth(srcWidth.toInt(), srcHeight.toInt());
+          finalImage = await picture.toImage(targetWidth, targetHeight);
```

Comment:
the original replay is equivalent to screenshot "full" quality AFAICT. I think we can integrate the configs and keep the replay default to "full" for now until we implement quality settings throughout the whole stack for replay.